Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -474,6 +474,7 @@ class WebViewImpl final : public CanMakeWeakPtr<WebViewImpl>, public CanMakeChec
     void preferencesDidChange();
 
     void setTextIndicator(WebCore::TextIndicator&, WebCore::TextIndicatorLifetime = WebCore::TextIndicatorLifetime::Permanent);
+    void updateTextIndicator(WebCore::TextIndicator&);
```

Comment:
Is there a reason this is a non-const ref? (I think it's just plumbed into the text indicator window as `Ref<TextIndicator>&&`, so I think that `const WebCore::TextIndicator&` should be okay.

Reply:
Because it’s following setTextIndicator above and that is how it is passed